vortexasdk/api/geography.py

```diff
@@ -25,10 +25,10 @@ class Geography(Node):
 
     id: ID
     name: Optional[str] = None
-    layer: Optional[List[str]] = None
+    layer: List[str] = []
     exclusion_rule: Optional[List[IDNameLayer]] = None
     hierarchy: Optional[List[IDLayer]] = None
-    pos: Optional[List[str]] = None
+    pos: Optional[List[str | float]] = None
```

Comment:
how come this can be both a string and float? it's a bit of an odd signature, no?